src/pages/Main/components/ShopList/ShopList.js

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import styled from 'styled-components';
+import Shop from '../Shop/Shop';
+
+const ShopList = ({ shopList }) => {
+  return (
+    <Container>
+      <Filter>
+        <Brand>
+          <h2>브랜드</h2>
+          <BrandList>
+            <ul>
+              <li>Nike</li>
+              <li>Adidas</li>
+              <li>New balanc</li>
+            </ul>
+          </BrandList>
+        </Brand>
+
+        <Size>
+          <h2>사이즈</h2>
+          <SizeList>
+            <button>230</button>
+            <button>240</button>
+            <button>250</button>
+            <button>260</button>
+            <button>270</button>
+          </SizeList>
+        </Size>
+      </Filter>
+      <Wrap>
+        <Products>
+          {shopList.map(item => {
+            return (
+              <Shop
+                key={item.id}
+                id={item.id}
+                img={item.img}
+                type={item.type}
+                subType={item.subType}
+                name={item.name}
+                number={item.number}
+              />
```

Comment:
```suggestion
              <Shop
                key={item.id}
                item={item}
              />
```
데이터 전체를 보내준 후 자식 컴포넌트에서는 구조분해할당 후 사용하면 더 깔끔합니다!